source/includes/fact-connect-intro.rst

```diff
@@ -1,7 +1,9 @@
 The :ref:`mongosync <c2c-mongosync>` utility creates a connection
 between MongoDB clusters. ``mongosync`` can make a connection between:
 
-- :ref:`self-managed clusters <server-replica-set-deploy-all>`,
-- :atlas:`MongoDB Atlas </>` hosted clusters, or 
-- a self-managed cluster and an Atlas hosted cluster.
-
+- :ref:`self-managed clusters <server-replica-set-deploy-all>`
+- :atlas:`MongoDB Atlas </>` hosted clusters
+- a self-managed cluster and an Atlas hosted cluster
```

Comment:
Is this supposed of end with `, or`?

Reply:
No, that contravenes our doc standards. Will add text before bullets.